app/api/private-cloud/products/[licencePlate]/comments/_operations/update.ts

```diff
@@ -1,3 +1,37 @@
-export default async function updateOp() {
-  return { message: 'Update comment successful' };
+import prisma from '@/core/prisma';
+
+interface UpdateCommentParams {
+  licencePlate: string;
+  commentId: string;
+  text: string;
+}
+
+export async function updateOp({ licencePlate, commentId, text }: UpdateCommentParams) {
```

Comment:
it takes an object as the argument type while others takes multiple arguments; better to have a consistency.